pages/mesosphere/dcos/2.0/release-notes/2.0.6/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 2.0.6
+title: Release notes for 2.0.6
+menuWeight: 0
+excerpt: Release notes for DC/OS 2.0.6, including Open Source attribution, and version policy.
+---
+DC/OS&trade; 2.0.6 was released on ????, 2020.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/2.0.6/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://downloads.mesosphere.com/dcos-enterprise/stable/2.0.6/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+New customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+# Release Summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+# DC/OS 
+
+## Components
+
+DC/OS 2.0.6 includes the following component versions:
+
+- ????
```

Comment:
?? From the DC/OS changes.md file: (https://github.com/dcos/dcos/blob/2.0.6/CHANGES.md)

Updated DC/OS UI to v5.1.7.
Update to Fluentbit 1.4.6